### DISCUSSION TITLE
Incorporating a Pressure Drop Calculation

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#25010] [Q&A Modules: General]
I am using a pre-made pressure drop calculator, but I am getting the following error: "A 'PressureDrop' is not a registered object." . Will I have to update my version of MOOSE to use this object?

**Comment by GiudGiud** [comment]
Hello

Maybe if your version of moose is really old. I added this object like 6 months ago or so

it s in the navier stokes module so you need to either use that module s executable or enable it in your application makefile 

Guillaume
